pennylane/_device.py

```diff
@@ -472,7 +498,7 @@ def expval(self, observable, wires, par):
 
         Args:
             observable (str or list[str]): name of the observable(s)
-            wires (List[int] or List[List[int]]): subsystems the observable(s) is to be measured on
+            wires (Wires): wires the observable(s) is to be measured on
```

Comment:
Now expval, var and sample take ``Wires`` arguments, and the devices have to translate to registers...